tests/pub-external-links/EPUB/package.opf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Publication Resources</dc:coverage>
+    <dc:creator>Zheng Xu</dc:creator>
+    <dc:date>2022-03-06</dc:date>
+    <dc:description>
+        Reading Systems SHOULD open links that resolve outside the EPUB Publication in a new browser instance
+        to ensure that the security and privacy controls of the browser are available to users.
```

Comment:
To match the style of other examples at https://w3c.github.io/epub-tests/#sec-publication-resources-data, I'd say something like this:

The XHTML content document contains a link to an external website. The reading system should open it in a new browser instance.

Reply:
thanks. Let me make the change.